README.md

```diff
@@ -1,3 +1,4 @@
 # hello-world
 This is my first repository and I created it to learn
 This what I changed when Isabella arrived from school
+Now we are trying to review what I have learnt 
```

Comment:
I think it will be better to replace 
are trying to review 
with "reviewing" :)